model/project_ref.go

```diff
@@ -43,7 +43,8 @@ type ProjectRef struct {
 
 	// The "Alerts" field is a map of trigger (e.g. 'task-failed') to
 	// the set of alert deliveries to be processed for that trigger.
-	Alerts map[string][]AlertConfig `bson:"alert_settings" json:"alert_config,omitempty"`
+	Alerts              map[string][]AlertConfig `bson:"alert_settings" json:"alert_config,omitempty"`
+	AlertOnBuildFailure bool                     `bson:"alert_on_failure" json:"alert_on_failure"`
```

Comment:
I've been using the words notify and notification to refer to all the event driven notifications feature. That said, i've been someone inconsistent with it. I don't really care either way, but i'd appreciate a line break/comment to separate this from Alerts above it (Which will be removed at the end of this EPIC)

Reply:
changed